docs/content/6.overlays/1.modal.md

```diff
@@ -178,6 +178,34 @@ defineShortcuts({
 </script>
 ```
 
+### Fullscreen
+
+Set the `fullscreen` prop to `true` to enable it.
+
+::component-example
+#default
+:modal-example-fullscreen
+
+#code
+```vue
+<script setup>
+const isOpen = ref(false)
+</script>
+
+<template>
+  <div>
+    <UButton label="Open" @click="isOpen = true" />
+
+    <UModal v-model="isOpen" :fullscreen="true">
+      <div class="p-4 flex items-center justify-center h-full w-full">
```

Comment:
I guess for this example, you could put the real code of the example, the one provided is a bit confusing in my opinion:

Reply:
Sure